scripts/ios_dump.sh

```diff
@@ -63,12 +66,13 @@ sed -i -e 's/<\/data>/<\/string>/g' $3
 # if fails (so in that case not jailbroken -- or 'not sure' for false negative).
 rm -rf /tmp/phonescanmnt
 mkdir -p /tmp/phonescanmnt
-"${libi}/ifuse" --root /tmp/phonescanmnt &> $4
+#"${libi}/ifuse" --root /tmp/phonescanmnt &> $4
 #"${libi}/ifuse" -u "$serial" --root /tmp/phonescanmnt &> $4
+ifuse -u "$serial" --root /tmp/phonescanmnt &> $4
 
 #lsof -ti tcp:2222 | xargs kill
-"${libi}/iproxy" 2222 22 & 
-"${DIR}/ios_ssh_expect.sh" localhost
+#"${libi}/iproxy" 2222 22 & 
```

Comment:
Don't comment the old line, just replace them with the new one. Git will keep the history if someone wants to go back. Duplicate lines mess up the `git grep` and also make the code ugly.

Reply:
Got it! Removed the commented lines in the latest commit.